src/crypto/hash/forge_sha1.js

```diff
@@ -0,0 +1,284 @@
+/**
```

Comment:
We no longer use forge. It has been replaced by Rusha and asmCrypto on the current master.